File: src/main/java/module/matches/statistics/MatchesOverviewColumn.java
package module.matches.statistics;

import core.gui.comp.table.UserColumn;

public class MatchesOverviewColumn extends UserColumn {

	protected MatchesOverviewColumn(int id, String name) {
		super(id, name);
		setDisplay(true);
	}

	protected MatchesOverviewColumn(int id,String name, String tooltip,int minWidth){
		super(id,name,tooltip);
		this.minWidth = minWidth;
		preferredWidth = minWidth;
	}
	
	
}

File: src/main/java/module/matches/statistics/MatchesOverviewColumn.kt
package module.matches.statistics

import core.gui.comp.table.UserColumn

class MatchesOverviewColumn : UserColumn {
    protected constructor(id: Int, name: String?) : super(id, name) {
        setDisplay(true)
    }

    constructor(id: Int, name: String?, tooltip: String?, minWidth: Int) : super(id, name, tooltip) {
        this.minWidth = minWidth
        preferredWidth = minWidth
    }
}
